server/game/cards/01_SOR/units/SuperlaserTechnician.ts

```diff
@@ -14,11 +13,7 @@ export default class SuperlaserTechnician extends NonLeaderUnitCard {
         this.addWhenDefeatedAbility({
             title: 'Put Superlaser Technician into play as a resource and ready it',
             optional: true,
-            // TODO: figure out how to change this from sequential to simultaneous while keeing the ready effect
-            immediateEffect: AbilityHelper.immediateEffects.sequential([
-                // TODO: create a MoveCardToResourceSystem and accompanying function in immediateEffects
-                AbilityHelper.immediateEffects.moveCard((context) => ({ target: context.source, destination: Location.Resource })),
-                AbilityHelper.immediateEffects.ready((context) => ({ target: context.source }))])
+            immediateEffect: AbilityHelper.immediateEffects.resourceCard({ target: this, readyResource: true })
```

Comment:
`AbilityHelper.immediateEffects.resourceCard((context) => ({ target: context.source, readyResource: true })))`

Reply:
This one pains me. Saying this there is so much clearer. Otherwise we're setting target to context.source, which is super nebulous